apps/fabric-website/src/pages/PageTemplates/TemplatePage/TemplatePage.doc.ts

```diff
@@ -54,7 +54,7 @@ export const TemplatePageProps: TFabricPlatformPageProps = {
     contact: require('!raw-loader!@uifabric/fabric-website/src/pages/PageTemplates/TemplatePage/docs/default/TemplateContact.md') as string,
 
     // Raw import of a types definition file to render the Implementation Section. (Array)
-    propertiesTablesSources: [require('!raw-loader!@uifabric/example-app-base/src/components/Page/Page.types.ts') as string]
+    propertiesTablesSources: [require('!raw-loader!office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.types.ts') as string]
```

Comment:
Not sure what's going on here. What are you trying to do?

Reply:
This is the page template for contributing pages to the site, copied over from the HIG. The previous import was pulling a types file from example-app-base which doesn't include them in the npm package. Since I'm working on getting the HIG updated with the new changes and importing components and pages from the public site, this import is causing a compile error as the file doesn't exist outside the OUFR repo.